theano/gpuarray/opt.py

```diff
@@ -1209,7 +1206,7 @@ def local_gpua_dot22scalar(op, context_name, inputs, outputs):
 @op_lifter([tensor.basic.Eye])
 @register_opt2([tensor.basic.Eye], 'fast_compile')
 def local_gpua_eye(op, context_name, inputs, outputs):
-    return GpuEye(dtype=op.dtype, context_name=context_name)
+    return [GpuEye(dtype=op.dtype, context_name=context_name)(*inputs)]
```

Comment:
Didn't make any difference. But since all the optimizers did this way, i followed that standard. Should i revert ?